master/getting-started/kubernetes/tutorials/stars-policy/index.md

```diff
@@ -31,7 +31,7 @@ kubectl get pods --all-namespaces --watch
 The management UI runs as a `NodePort` Service on Kubernetes, and shows the connectivity
 of the Services in this example.
 
-If you're running the vagrant cluster, you can view the UI by visiting `http://172.18.18.102:30002` in a browser.  For other clusters, accessing
+If you're running the vagrant cluster, you can view the UI by visiting `http://<vagrant-ip>:30002` in a browser.  For other clusters, accessing
```

Comment:
I think instead of bleeping out these IPs we should settle on a standard IP range to use in demos for hosts and a standard range for containers. 

I don't like all these `<ip>:<port>` templating we stick all over our docs.